.github/workflows/create-backport-trackers.yml

```diff
@@ -0,0 +1,15 @@
+---
+name: Create backport trackers for trackers in "needs-backport" state
+on:
+  schedule:
+    # Every 5 minutes: https://crontab.guru/every-5-minutes
+    - cron: '*/5 * * * *'
+env:
+  BACKPORT_BOT_REDMINE_API_KEY: ${{ secrets.BACKPORT_BOT_REDMINE_API_KEY }}
+  
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Look for trackers in "needs-backport" state and create the corresponding backport trackers
+        run: echo "Replace this line with the Backport bot script invokation"
```

Comment:
I'm assuming this is here so you can test the workflow first and will file a separate PR that actually runs the script?

Reply:
Yup, the final version is here!